build.sh

```diff
@@ -184,6 +192,7 @@ openslidejava_upurl="https://github.com/openslide/openslide-java/tags"
 
 # Update-checking regexes
 zlib_upregex="source code, version ([0-9.]+)"
+libzip_upregex="libzip-([0-9]+\.[0-9]+\.[0-9]+)\.tar"
```

Comment:
There have been 1.0 and 1.1 releases.  This regex would detect neither.  Don't try to count components in the version number; just use `([0-9.]+)` like the other regexes.